atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -685,58 +712,78 @@ private void doTestGetRangePaging(int numColumnsInMetadata, int batchSizeHint, b
         TableReference tableRef = createTableWithNamedColumns(numColumnsInMetadata);
 
         Map<Cell, byte[]> values = new HashMap<Cell, byte[]>();
-        values.put(Cell.create("00".getBytes(), "c1".getBytes()), "a".getBytes());
-        values.put(Cell.create("00".getBytes(), "c2".getBytes()), "b".getBytes());
-
-        values.put(Cell.create("01".getBytes(), RangeRequests.getFirstRowName()), "c".getBytes());
-
-        values.put(Cell.create("02".getBytes(), "c1".getBytes()), "d".getBytes());
-        values.put(Cell.create("02".getBytes(), "c2".getBytes()), "e".getBytes());
-
-        values.put(Cell.create("03".getBytes(), "c1".getBytes()), "f".getBytes());
-
-        values.put(Cell.create("04".getBytes(), "c1".getBytes()), "g".getBytes());
-        values.put(Cell.create("04".getBytes(), RangeRequests.getLastRowName()), "h".getBytes());
-
-        values.put(Cell.create("05".getBytes(), "c1".getBytes()), "i".getBytes());
-        values.put(Cell.create(RangeRequests.getLastRowName(), "c1".getBytes()), "j".getBytes());
+        values.put(Cell.create("00".getBytes(StandardCharsets.UTF_8), "c1".getBytes(StandardCharsets.UTF_8)),
```

Comment:
Understand you're trying to minimise the diff here, but I think this and the changes at the start of the file would look a lot better if we just had a helper method in this class for `utf8(String s)`.